components/Search/SearchContent.tsx

```diff
@@ -10,44 +10,66 @@ import {
   ModalContent,
   ModalHeader,
   Box,
-} from '@chakra-ui/react'
-import { Scrollbars } from 'react-custom-scrollbars-2'
-import data from './searchdata.json'
-import { SearchResultItem } from './SearchResultItem'
-import { ThemeContext } from '../../util/themecontext'
+} from '@chakra-ui/react';
+import { Scrollbars } from 'react-custom-scrollbars-2';
+import data from './searchdata.json';
+import { SearchResultItem } from './SearchResultItem';
+import { ThemeContext } from '../../util/themecontext';
+import { useDebounce } from './useDebounce'; // Import the debounce hook
+import fuseIndex from './fuse-index.json'; // Static import of the precomputed index
```

Comment:
Even without generating the index, please commit the following empty index file to prevent this section from causing an error:

```json
{
  "keys": [],
  "records": []
}
```

Reply:
Thanks for the feedback! I will add an empty index.